### DISCUSSION TITLE
Building on Fedora 32

### DISCUSSION THREAD

**Discussion by ctmattice** [geldata/gel#1509] [Q&A] [answered]
A couple of days ago I was able to build edgedb from source but had some warnings both on edb test and starting the server. Tried again this mourning to build with the same errors.

On edb test command I get

================================================================================
WARNING: test_edgeql_syntax_ddl_function_49 (test_edgeql_syntax.TestEdgeQLParser)
--------------------------------------------------------------------------------
/usr/lib64/python3.8/unittest/case.py:633: RuntimeWarning: coroutine 'TestEdgeQLParser.test_edgeql_syntax_ddl_function_49' was never awaited
  method()

I can start the server and everything appears ok, pull up another shell, enter the egdedb and it errors with

Can't decode param b"pgaddr": invalid type: integer `64181`, expected a string at line 1 column 49
EdgeDB 1.0-alpha.4+dev.28.gd81cf850

Any idea what I might be missing?

**Comment by 1st1** [comment]
Re warning: it's a bug, the test shouldn't be decorated as asynchronous. 

The second exception was recently fixed, IIRC. 

cc @tailhook

**Reply by tailhook** [reply]
The warning for test will be fixed by #1498

The warning of `pgaddr` is because you use old edgedb CLI tools. You should be using CLI tools from master/nightly for edgedb from master. setup.py installs nightly CLI tools automatically, but you probably have some other version installed in $PATH (check with `which edgedb` in shell). You can also disregard this warning unless you're using `\psql` command.

**Comment by ctmattice** [comment]
ok thanks, I have been building from github master, last clone was around 10:30EST this morning. when I start the server it reports EdgeDB 1.0-alpha.4+dev.28.gd81cf850

which edgedb list /usr/bin/edgedb

**Reply by tailhook** [reply]
> ok thanks, I have been building from github master, last clone was around 10:30EST this morning. when I start the server it reports EdgeDB 1.0-alpha.4+dev.28.gd81cf850

The aforementioned fix is not yet in master, but nevermind it doesn't break anything.

> which edgedb list /usr/bin/edgedb

This is definitely a separately installed tool. Just reinstall the CLI tool from nightly. Did you install a package?

Or you can use CLI tool at `./build/cli/bin/edgedb` in your edgedb build.

**Comment by ctmattice** [comment, marked as answer]
Honestly, I don't remember on CLI tool. I doubt it was a package, might be remnants of a prior failed build. Only deleted and re-initiated the project directory

I can use ./build/cli/bin/edgedb and error is gone
